libs/brackets-server/embedded-ext/wasm-debug/nls/root/strings.js

```diff
@@ -0,0 +1,6 @@
+// English - root strings
+define({
+    "ERROR_TRANSLATE_WASM"    : "Failed to translate wasm!",
+    "ERROR_NOT_FOUND_FILE"    : "Not found selected file",
```

Comment:
check english for "Not found selected file" ?

Reply:
Fixed.